src/object.rs

```diff
@@ -1949,6 +2034,88 @@ impl<T: ObjectType> ObjectExt for T {
     }
 }
 
+// Validate that the given property value has an acceptable type for the given property pspec
+// and if necessary update the value
+fn validate_property_type(
```

Comment:
validate_and_set_property ? Then why not just set_property_value?
`and if necessary update the value` -> `if acceptable` ?

Reply:
Or comment is `if necessary update the value type` ?

Reply:
I don't understand the comment. This is only validating, not setting the property. Because the same function is a) also used to validate in `Object::new()` (where we can't set the property ourselves because there is no object yet), and b) in `ObjectExt::set_properties()` where we first want to validate all properties before setting one of them.

What exactly do you want changed here? :)

Reply:
Seems I want only update commentary, that it updated only type of value, not value itself

Reply:
and doing it almost always, without "necessary"

Reply:
Sorry, it changes really only if "necessary"

Reply:
It's rarely updating the type. Only if the given type is a subclass of the required type instead of the type itself, and if `g_param_value_validate()` is doing changes, which also shouldn't happen often.

Can you suggest a wording that makes it more clear for you?

Reply:
"if necessary update the type of value" or something like this

Reply:
`g_param_value_validate()` can also change the value of the value

Reply:
Oh, this is bad :( Then current comment is good.